src/us/kbase/workspace/kbase/LocalTypeServerMethods.java

```diff
@@ -114,12 +115,12 @@ public Map<String,String> registerTypespec(
 		final Map<TypeDefName, TypeChange> res;
 		if (params.getMod() != null) {
 			 res = types.compileTypeSpec(
-					getUser(authPart), params.getMod(),
+					getUser(token), params.getMod(),
```

Comment:
will you add a test for this?

Reply:
Not right now. This is the same code we talked about before, where I just moved it from the server class into this class verbatim so I could write an alternate implementation.

Reply:
Got it, cool.